draft-kohbrok-mls-associated-parties.md

```diff
@@ -153,8 +115,23 @@ struct {
 } AssociatedParties
 ~~~
 
-A group member can add or remove associated parties in the context of a group by
-sending Add- or RemoveAssociatedParty proposals.
+## Associated parties as external senders
+
+Associated parties can act as external senders and thus send external proposals
+as specified in Section 12.1.8 of {{!RFC9420}}. An AP that sends an external
+proposal MUST use the `external` SenderType with an index that is the number of
+entries in the group's `external_senders` (0 if no such extension is present)
+plus its own index in the group's AssociatedParties extension. For external
+sender indices, the AssociatedParties extension thus extends the
+`external_senders` extension.
+
+Recipients of a message sent by an AP MUST verify the message's signature
+against the `signature_key` in the sender's AssociatedPartyEntry.
```

Comment:
I don't see that APs as external senders is strictly required. You could have the same entity in both the AP list and the external senders and everything would still work fine.

Reply:
Fair point. But then I'd want to mandate that APs are in the external senders list, because we want the HPKE public keys that we encrypt secrets to to be authenticated somehow.

Now that I think about it, having them in the external senders is in fact a much nicer approach. I'll change it.